app/helpers/application_helper.rb

```diff
@@ -15,6 +15,10 @@ def self.humanize_cpf(cpf)
     "#{cpf[0..2]}.#{cpf[3..5]}.#{cpf[6..8]}-#{cpf[9..11]}"
   end
 
+  def humanize_phone_number(phone)
+    phone.to_s.gsub(/(\d{2})(\d{4,5})(\d{4})/, '(\1) \2-\3')
+  end
```

Comment:
I wouldn't even go this far, I believe it's totally fine to have the already humanized phone on the config file -- as this will not work for toll free numbers for instance (0800). It's a nice touch though!